OsmAnd/res/values/strings.xml

```diff
@@ -318,33 +318,33 @@
     <string name="magnetic_bearing_widget_desc">Angle between magnetic north and the target point observed from your location.</string>
     <string name="time_to_go_desc">Time in minutes or hours needed to reach your destination.</string>
     <string name="arrival_time_widget_desc">Time of arrival at the destination point.</string>
-    <string name="lanes_widgets_desc">Shows the current road lanes layout and highlights the ones to take while navigating.</string>
-    <string name="map_markers_bar_widget_desc">The top bar widget displays the distance and direction to the next Map marker from your current location. The widget can be configured to show 1 or 2 markers.</string>
-    <string name="street_name_widget_desc">The Street name widget shows the current street name with a straight location arrow, or the next street name with an arrow indicating an upcoming turn.</string>
-    <string name="coordinates_widget_desc">Shows the geographic coordinates of current geolocation.</string>
+    <string name="lanes_widgets_desc">Highlights the road lane to take while navigating and shows the layout.</string>
+    <string name="map_markers_bar_widget_desc">On the top bar widget you could see the distance and the direction to the marker from current location. It is possible to show 1 or 2 top markers.</string>
+    <string name="street_name_widget_desc">Shows the current street name with a straight location arrow or next street name with the maneuver to complete.</string>
+    <string name="coordinates_widget_desc">It shows the geographic coordinates of current geo-location.</string>
     <string name="shared_string_widget">Widget</string>
     <string name="av_def_action_choose">On request</string>
-    <string name="audio_video_notes_desc">Provides quick access to start/stop taking an audio, video, or photo note.</string>
-    <string name="bearing_desc">Shows the absolute or relative bearing to a target in angular units (in 180 degree, 360 degree, or milliradians mode).</string>
-    <string name="navigation_points_desc">Navigational widgets are enabled during navigation to display information such as distance, arrival or remaining time, next turns, bearing, current/next street name, lane information, max speed, approaching alerts, POIs, waypoints.</string>
+    <string name="audio_video_notes_desc">Provides quick access to start/stop taking a recording (audio, video or photo).</string>
+    <string name="bearing_desc">The bearing widget shows the absolute or relative bearing to a target in angular units (degrees 180, degrees 360, milliradians).</string>
```

Comment:
So why here the other way round: Why re-introduce the "widget" rather than going with "Shows" like you corrected elsewhere?